core/crawler-core/src/test/java/org/eclipse/edc/catalog/cache/ExecutionManagerTest.java

```diff
@@ -170,9 +172,38 @@ void executePlan_withCustomFiltering() {
         verify(filter, times(3)).test(any());
     }
 
+    @Test
+    void shutdownPlan_shouldNotStopPlanWhenGloballyDisabled() {
+        manager = createManagerBuilder().isEnabled(false).build();
+        var mockPlan = mock(ExecutionPlan.class);
+        manager.shutdownPlan(mockPlan);
+
+        verify(monitorMock).warning(eq("Execution of crawlers is globally disabled."));
```

Comment:
not the biggest fan of verifying concrete log messages. better to do 
```java
verify(monitorMock).warning(anyString());
```
this will prevent the test from breaking if the log line changes, which does not change the behaviour.